### ISSUE TITLE
Is there any suitable alternative?

### ISSUE THREAD

**Issue by nodew** [dagrejs/dagre#318]
As the repo is deprecated.

**Comment by danyflorez** [comment]
Maybe these?

https://github.com/kieler/elkjs

https://github.com/dhotson/springy

**Comment by alexec** [comment]
I don't think this repo is "deprecated" per-se - just not maintained. You could fork it.

**Comment by tensiondriven** [comment]
If anyone is doing a lot of active work on their fork, we could make a PR to add a link to that repo to the README so people could like to a more updated version, or perhaps the maintainer would reassign ownership to someone else?

**Comment by georgefst** [comment]
Is there any discussion anywhere about why this project was "deprecated"? It seemingly happened in [July last year](https://github.com/dagrejs/dagre/commit/c8bb4a1b891fc50071e6fac7bd84658d31eb9d8a) with no announcement.

Regardless, I agree with @alexec that "deprecated" seems the wrong word here. The library is still perfectly usable, and has no clear successor. I'd like to think that dagre is "temporarily lacking maintainers", rather than truly deprecated. Hopefully @dagrejs are genuinely open to contact from those who "are qualified and have the resources to make a serious commitment to fully take over ownership of the project", and someone will step up. For one thing, people are still building things on top of it ([1](https://twitter.com/segphault/status/1493284689905393670), [2](https://reactflow.dev/docs/examples/layout/dagre/)).

**Comment by curran** [comment]
Check out these NPM weekly download numbers. Quite interesting. Dagre has _way_ more than elkjs.

# dagre

 * https://www.npmjs.com/package/dagre

![image](https://user-images.githubusercontent.com/68416/208209247-bb6d6b2a-7650-4ead-bcb0-00a47193dec7.png)

# elkjs

 * https://www.npmjs.com/package/elkjs

![image](https://user-images.githubusercontent.com/68416/208209370-91493051-dc44-429f-a5da-3f9b799b894a.png)

# webcola

 * https://www.npmjs.com/package/webcola

![image](https://user-images.githubusercontent.com/68416/208209495-e47e2586-278f-4028-8149-025895ad2480.png)
